lib/livebook_web/live/session_live.ex

```diff
@@ -672,27 +703,32 @@ defmodule LivebookWeb.SessionLive do
         %{"cell_id" => cell_id, "tag" => tag, "revision" => revision},
         socket
       ) do
+    assert_policy!(socket, :edit)
```

Comment:
@jonatanklosko is this correct? It seems all clients would use this as read so we can clean space in the server if necessary.

Reply:
Correct, should be `:read`!